test/format/format.h

```diff
@@ -34,34 +34,23 @@
 #include <signal.h>
 
 #define EXTPATH "../../ext/" /* Extensions path */
-
-#ifndef LZ4_PATH
-#define LZ4_PATH EXTPATH "compressors/lz4/.libs/libwiredtiger_lz4.so"
+#ifndef EXT_LIBPATH
+#define EXT_LIBPATH ".libs/"
 #endif
 
-#ifndef SNAPPY_PATH
-#define SNAPPY_PATH EXTPATH "compressors/snappy/.libs/libwiredtiger_snappy.so"
-#endif
+#define LZ4_PATH EXTPATH "compressors/lz4/" EXT_LIBPATH "libwiredtiger_lz4.so"
```

Comment:
Don't we want to keep the `ifndef` statements?

Reply:
We shouldn't need to. I originally added the `ifndef` statements when I initially ported `test/format` to CMake some time ago, mainly as a way to override the values of `LZ4_PATH`, `SNAPPY_PATH` etc. Though that's no longer the case, as we now just need to pass in `EXT_LIBPATH` (removing the `ifndef` returns it back to how it was originally).